framework/web_app/src/boot/axios.ts

```diff
@@ -1,11 +1,11 @@
-import { boot } from 'quasar/wrappers';
 import axios, { AxiosInstance } from 'axios';
```

Comment:
we can move the axios stuff to here possibly?

this is up for grabs if you want

Reply:
![image](https://github.com/BenTalese/DiscountDora/assets/95552720/c1c4a438-12f4-4eb6-8cf3-9f2363988cfe)

Seems it is not the spot for the axios endpoint, rather for configuration.
We can move any axios before/after request middleware as request or response interceptors here.

Reply:
opting to delete this file as we're handling axios via other means